app/survey_config/survey_type.py

```diff
@@ -0,0 +1,11 @@
+from enum import Enum
+
+
+class SurveyType(Enum):
+    business = "business"
+    social = "social"
+    default = "default"
+    health = "health"
+    northernireland = "northernireland"
+    census = "census"
+    census_nisra = "census-nisra"
```

Comment:
Normally constants are uppercase. No hard and fast rule but worth being consistent. They are uppercase in validator so worth doing the same here. Also, there's this: https://mail.python.org/pipermail/python-ideas/2016-September/042340.html

Reply:
All of these changed now, thanks!